invenio_records_resources/services/files/tasks.py

```diff
@@ -22,7 +22,8 @@ def fetch_file(service_id, record_id, file_key):
     file_record = service.read_file_metadata(system_identity, record_id, file_key)
     source_url = file_record.data["uri"]
     # download file
-    with requests.get(source_url, stream=True) as response:
+    # verify=True for self signed certificates by default
+    with requests.get(source_url, stream=True, allow_redirects=True) as response:
```

Comment:
why do we need the `allow_redirects` here? what was the problem?

Reply:
It's more of making sure we don't have issues in case a remote file fetch URL relies on redirects (e.g. imagine CERNBox link redirecting to the new version). It doesn't affect URLs that directly serve the file content, but makes sure we don't store the 3xx response as the file content.